controllers/dscinitialization/resources/observability/alertmanager/alertmanager-email.yaml

```diff
@@ -0,0 +1,617 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: alertmanager-rhoai
+  namespace: {{ .MonNamespace }}
+data:
+    email-rhoai-subject.tmpl: |
+        {{ if gt (len .Alerts.Firing) 1 }}
```

Comment:
@zdtsw  the bits having`{{` `}}` are problematic as the templating is trying to substitute those values 😅 . This is done by the features API so changing the delim will affect everything(so not a good idea )

Reply:
what if we move this logic into the code itself?

Reply:
this is the fix https://github.com/opendatahub-io/opendatahub-operator/pull/1004/commits/345b55b7ba539bfcfcaa3f2133fae8f30557b027

Reply:
great!!! will get those changes in